client/my-sites/checkout/composite-checkout/components/wp-checkout-order-summary.tsx

```diff
@@ -310,6 +312,12 @@ function CheckoutSummaryFeaturesList( props: {
 	const plans = responseCart.products.filter( ( product ) => isPlan( product ) );
 	const hasPlanInCart = plans.length > 0;
 
+	const jetpackProducts = responseCart.products.filter( ( product ) =>
+		isJetpackProduct( product )
+	);
```

Comment:
It just looks nice, but it's totally extra. 

```suggestion
	const jetpackProducts = responseCart.products.filter( isJetpackProduct );
```

Reply:
I totally spaced that we can do this 😅